Src/Support/GoogleApis.Core/Apis/Json/NewtonsoftJsonSerializer.cs

```diff
@@ -75,10 +76,13 @@ public static NewtonsoftJsonSerializer Instance
         static NewtonsoftJsonSerializer()
         {
             // Initialize the Newtonsoft serializer.
-            JsonSerializerSettings settings = new JsonSerializerSettings();
-            settings.NullValueHandling = NullValueHandling.Ignore;
-            settings.Converters.Add(new RFC3339DateTimeConverter());
-            newtonsoftSerializer = JsonSerializer.Create(settings);
+            s_settings = new JsonSerializerSettings

```

Comment:
There's a couple of reasons I'd rather keep the test there:
* The bug was triggered due to the generated library; it provides the repro.
* Having a test on a generated library is a good thing.